api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -42,16 +42,20 @@ default Map<String, DbUser> getUsersMappedByUsernames(Collection<String> usernam
   List<DbUser> findUsers();
 
   @Query(
-      "SELECT user FROM DbUser user JOIN FETCH user.userInitialCreditsExpiration uice WHERE uice.expirationCleanupTime IS NULL")
```

Comment:
I tried keeping the JOIN FETCH, but simply changing to user.userId caused Spring errors.  Switching to regular JOIN fixed it.

Reply:
I think that since this is just returning user ids and not user objects, the JOIN seems more appropriate.